crates/samples/appwindow/src/main.rs

```diff
@@ -0,0 +1,93 @@
+use ::windows::core::*;
+use ::windows_app::Microsoft::UI::Windowing::*;
+use ::windows_app::*;
+use windows::Foundation::TypedEventHandler;
+use windows::Win32::Foundation::{BOOL, HWND, RECT};
+use windows::Win32::UI::HiDpi::GetDpiForWindow;
+use windows::Win32::UI::WindowsAndMessaging::{
+    DispatchMessageW, GetMessageW, GetSystemMetrics, GetWindowRect, PostQuitMessage, SetWindowPos,
+    TranslateMessage, MSG, SM_CXSCREEN, SM_CYSCREEN, SWP_NOMOVE, SWP_NOSIZE,
+};
+
+fn main() -> Result<()> {
+    bootstrap::initialize()?;
+    sample_main()?;
+    bootstrap::uninitialize()
+}
```

Comment:
```suggestion
fn main() -> Result<()> {
    sample_main()?;
}
```

Why not this?

Reply:
As soon as WinAppSdk 1.1 ships, definitely!